errors.go

```diff
@@ -25,6 +25,10 @@ const ErrPause = errorType("pause channel")
 // use to resume the channel
 const ErrResume = errorType("resume channel")
 
+// ErrNoOp is a special error that the RequestReceived / ResponseReceived hooks can
+// use to neither resume nor pause the channel but maintain the status quo.
+const ErrNoOp = errorType("no-op")
+
```

Comment:
Instead of introducing ErrNoOp, would it make sense to change the code to
- explicitly return ErrResume when the revalidator wants to resume the transfer
- return nil when the revalidator wants a NoOp

Reply:
@dirkmc 

I thought of that but doing that breaks a whole bunch of tests in `go-data-transfer`. Happy to go down this path and fix the tests if you think that's more valuable.